src/transport/MMCollisionInt.cpp

```diff
@@ -428,8 +428,9 @@ void MMCollisionInt::fit_omega22(int degree, doublereal deltastar,
     w[0]= -1.0;
     double rmserr = polyfit(n, degree, logT, values.data(), w.data(), o22);
     if (m_loglevel > 0 && rmserr > 0.01) {
-        writelogf("Warning: RMS error = %12.6g in omega_22 fit"
-                  "with delta* = %12.6g\n", rmserr, deltastar);
+        warn_user("MMCollisionInt::fit_omega22",
+            "RMS error = {:12.6g} in omega_22 fit"
```

Comment:
```suggestion
            "RMS error = {:12.6g} in omega_22 fit "
```
(I realize that this was already missing a space)